server/server.go

```diff
@@ -70,31 +70,12 @@ func NewServer(ctx *Context) (*Server, error) {
 	if ctx == nil {
 		return nil, util.Error("ctx must not be null")
 	}
-	// Determine hostname in case it hasn't been specified in -rpc or -http.
+	// Determine hostname in case it hasn't been specified in -http.
 	host, err := os.Hostname()
```

Comment:
This was used before to construct an address that other nodes in the cluster would be able to connect to. If we start a local RPC server with :8080 as the address, will we end up with rpc.Server.Addr() returning something that makes sense? Keep in mind that whatever address is returned from rpc.Server.Addr() is what will be gossiped so that other nodes may connect directly. The idea before with figuring out the host here was to allow --rpc=:9000 and prefix the :9000 with a hostname which hopefully would be resolvable.